### PR TITLE
No amp4test log on travis.

### PR DESCRIPTION
**Pull request by lannka**
Still want logs on local testing.

### CODE REVIEW

**Comment by lannka** [comment]
I think it's fine? I assume running test locally is most in dev / debug mode?
(I haven't been running all tests locally for years)

**Comment by rsimha** [comment]
Running all tests locally is a routine workflow for the infra team, so these logs do come in the way due to how often they're printed.

A couple suggestions:
1. Change the `AMP_TEST` var in `runtime-test.js` to something like `ENABLE_A4A_LOGGING`, and enable it only when a `--a4a-logging` flag is passed in.
2. Write the logs to a local file (added to `.gitignore`) instead of to the console, and verify that the logs are present.

**Comment by lannka** [comment]
@rsimha instead of introducing another flag, what's your attitude to reusing the existing `--files=` flag? Assuming when tests run alone we are OK with verbal logs?

**Comment by rsimha** [comment]
Good idea, using `--files` sgtm.

**Comment by lannka** [comment]
@rsimha changed to read `--files`. PTAL

File: build-system/amp4test.js [hunk 1]
```diff
@@ -16,17 +16,21 @@
...
- * Logs the given messages to the console in local dev mode, but not while
- * running automated tests.
+ * Logs the given messages to the console when running individual test files
+ * with --files commend line argument.
  * @param {*} messages
  */
 function log(...messages) {
-  if (!process.env.AMP_TEST) {
-    console.log(messages);
+  if (argv.files) {
```
Comment (rsimha) [inline build-system/amp4test.js]: I just realized we also have a `verbose` flag. You could use that instead of (or in addition to) `files` if you'd like.

Comment (lannka) [reply build-system/amp4test.js]: SG. switched to `--verbose`

**Pull request merged**